roles/container_runtime/tasks/common/pre.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Set use_crio to True if cri-o rpm is requested
```

Comment:
We should move this to init/basic_facts.yml because this code won't take effect during the install, just prerequisites.